src/test/java/core/basesyntax/dao/ActivityDaoDbImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivityDaoDbImplTest {
+
+    private static ActivityDaoDb activityDaoDb;
+    private static Fruit fruitFirst;
+    private static Fruit fruitSecond;
+
+    @BeforeClass
+    public static void beforeClass() {
```

Comment:
```suggestion
    @BeforeClass
    public static void setUp() {
```
the same goes for every other code like this

Reply:
ok